src/pages/projectSettings/tabs/utils/alias.ts

```diff
@@ -431,7 +431,7 @@ export const gitTagArray = {
         gitTag: gitTag.schema,
         specifier: {
           type: "string" as "string",
-          title: "Specify Via",
```

Comment:
was there a reason to remove this? did you want to remove this instance too? :0

<img width="507" alt="Screenshot 2023-05-31 at 11 32 04 AM" src="https://github.com/evergreen-ci/spruce/assets/47064971/8552f648-e88a-493d-aab7-afa8bc5d7050">

Reply:
Created EVG-20037 to make some improvements to these!